.github/workflows/goalie.yaml

```diff
@@ -1,8 +1,9 @@
 # on a review from someone in the reviewers group, remove the awaiting-review label and add the awaiting-author label
 name: 'goalie: run cron'
-on:
-  schedule:
-    - cron: '* * * * *'
+#on:
+#  schedule:
+#    - cron: '* * * * *'
```

Comment:
cron schedule no longer needed?

Reply:
It's just so I can run it as a test in the PR

Reply:
Will remove once this passes

Reply:
🧙